portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+var curSlide = 0; //by default, display first slide
```

Comment:
let's use 'const' and 'let' instead of var

Reply:
@megankj Small pun intended?